dipy/denoise/noise_estimate.py

```diff
@@ -117,7 +120,8 @@ def piesno(data, N, alpha=0.01, l=100, itermax=100, eps=1e-5, return_mask=False)
                                                           itermax=itermax,
                                                           eps=eps,
                                                           return_mask=True,
-                                                          initial_estimation=initial_estimation)
```

Comment:
And agreed with @samuelstjean on not breaking the line to move the parameter to the next line.